_includes/tutorials/finding-distinct/ksql/markup/dev/create-inputs.adoc

```diff
@@ -0,0 +1,5 @@
+To start off the implementation of this scenario, you need to create a stream that represents the clicks from the users. Since we will be handling time in this scenario, it is important that each click contains a timestamp indicating when that click was done. The field `TIMESTAMP` will be used for this purpose.
```

Comment:
```suggestion
To start off the implementation of this scenario, we will create a stream that represents the clicks from the users. Since we will be handling time, it is important that each click contains a timestamp indicating when that click was done. The field `TIMESTAMP` will be used for this purpose.
```